dangerfile.js

```diff
@@ -56,12 +56,19 @@ function generateMDTable(headers, body) {
  * @param {string[]} headers
  */
 function addPercent(change, includeEmoji) {
-  if (change > 0 && includeEmoji) {
-    return `:small_red_triangle:+${change}%`;
-  } else if (change > 0) {
-    return `+${change}%`;
+  if (!isFinite(change)) {
+    // When a new package is created
+    return 'n/a';
+  }
+  const formatted = (change * 100).toFixed(1);
+  if (/^-/.test(formatted)) {
+    return `${formatted}%`;
   } else {
-    return `${change}%`;
+    if (includeEmoji) {
```

Comment:
"No changes" becomes "+0.0%". Do you think maybe we could special case this and show `(none)` or `(same)`? Danger will skip over them but they still show up in local output. Or at least maybe lose the `+` if it's exact zero.

Reply:
ahh good catch, I noticed this in the original code and forgot to match it.